### DISCUSSION TITLE
tailwindCSS does not work with HBS files

### DISCUSSION THREAD

**Discussion by Mrahmani71** [tailwindlabs/tailwindcss#6633] [Help] [answered]
Hi, the styles I am using in the tags of the ".hbs" file do not apply. I think the classes I used in the ".hbs" file are not saved in the "output.css" file.

Does anyone have a solution?
MERCI

![help](https://user-images.githubusercontent.com/13013819/146691080-fe52cb8e-1bec-460a-971e-8e0e465b52fa.jpg)

**Comment by neupauer** [comment, marked as answer]
Hi @Mrahmani71, Tailwind looks for classes only in those files that are defined in the `content` section of your `tailwind.config.js`. So make sure that you have correct glob pattern in it.
Maybe something like this:
```js
module.exports = {
  content: [
    './pages/**/*.hbs',
  ]
}
```

For more info see [Tailwind CSS | Content Configuration](https://tailwindcss.com/docs/content-configuration).

**Reply by Mrahmani71** [reply]
Thank you, it worked
